deterministic-build.yml

```diff
@@ -0,0 +1,33 @@
+trigger:
+- master
+
+strategy:
+  matrix:
+    lin:
+      runtime: 'linux-x64'
+    mac:
+      runtime: 'osx-x64'
+    win:
+      runtime: 'win7-x64'
+
+pool:
+  vmImage: ubuntu-16.04
+
+steps:
+- task: UseDotNet@2
+  displayName: 'Install .NET Core 3'
+  inputs:
+    version: 3.1.x
+
+- task: Bash@3
+  displayName: 'Build and compare'
+  inputs:
+    targetType: 'inline'
+    script: |
+      cd WalletWasabi.Gui;
+      for i in `seq 2`;
+      do
+      dotnet clean --configuration Release && dotnet publish --configuration Release --force --output "./dist${i}" --self-contained true --runtime "$(runtime)" /p:VersionPrefix=1.1.20 --disable-parallel --no-cache /p:DebugType=none /p:DebugSymbols=false /p:ErrorReport=none /p:DocumentationFile=\"\" /p:Deterministic=true;
```

Comment:
`/p:VersionPrefix=1.1.20` - it may be better to put there something that denotes that this is a deterministic build test. Maybe `0.0.0` or something special like `99.99.99`?

Not sure whether one can actually put there only version numbers or any string. Does anyone know?

Reply:
There are `Version`, `VersionPrefix` and `VersionSuffix`. We are not specifying the version in the `.csproj` so we always put it explicitly as part of the `dotnet publish`. What I think we can do is to set the `VersionSuffix` here, it can be whatever you want.

Reply:
`VersionSuffix` sounds good.